Core/include/Acts/Utilities/AngleHelpers.hpp

```diff
@@ -9,26 +9,68 @@
 #pragma once
 
 #include <cmath>
+#include <numbers>
 
 namespace Acts::AngleHelpers {
 
+template <typename Scalar>
+struct EtaThetaConversionTraits {};
+
+template <>
+struct EtaThetaConversionTraits<float> {
+  static constexpr float minTheta = 1e-6f;
+  static constexpr float maxTheta = std::numbers::pi_v<float> - 1e-6f;
```

Comment:
It's not a big deal, but this might be a bit better:
```suggestion
  static constexpr float maxTheta = std::numbers::pi_v<float> - minTheta;
```
to be sure of typos.

Similarly for the other specialisation.

Reply:
sure - my thought was that the min/max would not have to match but as it is for now they do so we can just deduplicate the numbers